### PR TITLE
chore: update the readme to have useful links

### PR DESCRIPTION
**Pull request by jrainville**
The only missing point is adding the release blog, but it's not available yet.

For the "pretty picture", I used a screen from the Figma. I can easily update it. Let me know @benjthayer if you have a better idea (I can do it in another PR). 
This current image is nice because it shows a lot of the small things, but it has some things that don't exist, like the QR code scanner and the `X joined the channel` message at the top (it only exists in group chats).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+Desktop client for the [Status Network](https://status.app/) built with [Nim](https://nim-lang.org/) and [Qt](https://www.qt.io/)
 
-![https://github.com/status-im/nim-status-client/blob/master/screenshot.png](https://github.com/status-im/nim-status-client/blob/master/screenshot.png)
+#### [Our Website](https://status.app/)
+
+#### [Need Help?](https://status.app/help)
+
+#### [Build Instructions](https://zealous-polka-dc7.notion.site/Building-ca1db4fb3baf4f15bab8da717832b743?pvs=74)
+
+#### [You have an idea for a cool feature or improvement?](https://status-desktop.featureupvote.com/)
```
Comment (caybro) [inline README.md:11]: We have it directly at https://status.app/feature-upvote

**Review by caybro — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jrainville) [reply README.md:11]: Oooh, way fancier! I'll update

File: screenshot.png
Comment (igor-sirotin) [inline screenshot.png:1]: Maybe smth like this, which is used on the website?

https://res.cloudinary.com/dhgck7ebz/image/upload/f_auto,c_limit,w_3840,q_auto/Platforms/Screens/Desktop%20Screens/Communities/Communities

image

It's also a bit too "future" and have some non-implemented features, but people already thought of what's required to be in the picture 🤔

File: README.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Status-desktop
```
Comment (igor-sirotin) [inline README.md:1]:
```suggestion
# Status Desktop
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
 
-Desktop client for the [Status Network](https://statusnetwork.com/) built with [Nim](https://nim-lang.org/) and [Qt](https://www.qt.io/)
+Desktop client for the [Status Network](https://status.app/) built with [Nim](https://nim-lang.org/) and [Qt](https://www.qt.io/)
```
Comment (igor-sirotin) [inline README.md:3]: I wonder if `Status Network` is a correct terminology now. We use this for a different thing.

**Review by igor-sirotin — APPROVED** [review]
Minimalistic it is 😄

**Comment by jrainville** [comment]
> Minimalistic it is 😄 Thanks!

haha yeah. The Docs teams can do something prettier later, but at least, it makes our Readme usable :D

**Pull request merged**